### PR TITLE
Implement sysconf for CPU online count

### PR DESCRIPTION
**Pull request by Dennisbonke**

### CODE REVIEW

File: posix/subsystem/src/requests.cpp [hunk 1]
```diff
@@ -3483,6 +3484,55 @@ async::result<void> serveRequests(std::shared_ptr<Process> self,
...
+				resp.set_value(kernResp->num_cpu());
+
+				auto [send_resp] = co_await helix_ng::exchangeMsgs(
+					conversation,
+					helix_ng::sendBragiHeadOnly(resp, frg::stl_allocator{})
+				);
+
+				HEL_CHECK(send_resp.error());
+			} else {
+				// Not handled, bubble up EINVAL.
```
Comment (no92) [inline posix/subsystem/src/requests.cpp:3526]: Maybe we should log the unhandled sysconf request num?

File: posix/subsystem/src/requests.cpp [hunk 2]
```diff
@@ -3483,6 +3484,55 @@ async::result<void> serveRequests(std::shared_ptr<Process> self,
...
+				resp.set_error(managarm::posix::Errors::ILLEGAL_ARGUMENTS);
+
+				auto [send_resp] = co_await helix_ng::exchangeMsgs(
```
Comment (no92) [inline posix/subsystem/src/requests.cpp:3529]: The sending of the reply in both branches could be moved outside of the if-else-statement.

**Review by no92 — COMMENTED** [review]
Two small notes, other than that LGTM.

File: posix/subsystem/src/requests.cpp (hunk 1 above)
Comment (Dennisbonke) [reply posix/subsystem/src/requests.cpp:3526]: That is done in mlibc land, EINVAL will get handled by the main sysconf function, logging for default values and panicking for unimplemented ones.

File: posix/subsystem/src/requests.cpp (hunk 2 above)
Comment (Dennisbonke) [reply posix/subsystem/src/requests.cpp:3529]: True, will fix

**Review by no92 — APPROVED** [review]
LGTM

**Pull request merged**